Dockerfile

```
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.11-slim
+
+WORKDIR /task_app
```

@YanaRudyi it is more common to add a folder as build run env variable and then reuse it to put files there
```suggestion
ENV APP_DIR /app
WORKDIR /${APP_DIR}

COPY requirements.txt /${APP_DIR}/requirements.txt
RUN pip3 install -r /${APP_DIR}/requirements.txt
```